grudge/trace_pair.py

```diff
@@ -497,9 +497,12 @@ def cross_rank_trace_pairs(
                 num_tag = actx.comm_tag_to_mpi_tag.get(comm_tag)
 
             if num_tag is None:
-                raise ValueError("Encountered unknown symbolic tag "
-                        f"'{comm_tag}'. To make this symbolic tag work, "
-                        f"use 'grudge.array_context.MPIPyOpenCLArrayContext' and "
+                num_tag = hash(comm_tag)
```

Comment:
This is *not* correct, as described in https://github.com/illinois-ceesd/mirgecom/issues/617#issuecomment-1057082716. We *could* merge this temporarily (generously plastered with FIXMEs and with an issue filed) to avoid inflicting temporary changes on mirgecom while we work on a better fix that avoids burdening the user with putting together `comm_tag_to_mpi_tag`.